workshops/pcb_level_1/README.md

```diff
@@ -0,0 +1,182 @@
+---
+name: 'PCB Level Part 1: Microcontroller Design'
+description: 'Design an Arduino Nano-compatible microcontroller board'
+author: '@karmanyaahm'
+img: './7.png'
+---
+
+In this workshop, we will design an Arduino Nano-compatible microcontroller board.
+
+<details>
+<summary>Prerequisites</summary>
+If you are not already familiar with the basics of using an ECAD tool like EasyEDA or KiCAD, check out @maggie's workshop on designing a PCB Business card:
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/enMtMOgimm4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Recommend a KiCAD video???
+
+In addition, the rest of the workshop assumes you know basic electronics terms like:
+1. Resistor: A resistor limits the electric current that flows through a circuit. Resistance is the restriction of current.
+2. Capacitor: A capacitor stores charge, and thus energy. It's like a tiny battery that charges and discharges very quickly.
+</details>
+
+
+
+I am using: 
+<label><input name="viewSettings" type="radio" data-to-hide=".kicad-img" checked=true/> KiCAD</label>
+<label><input name="viewSettings" type="radio" data-to-hide=".easyeda-img" />EasyEDA</label>
+<script>
+    // this script will switch between KiCAD images and EasyEDA images based on radio buttons above
+
```

Comment:
This little bit of JS renders as:
![image](https://github.com/hackclub/hackclub/assets/32671690/8708ccb2-63d4-4370-b7c9-55f215ce9957)
and allows the reader to select between images for KiCAD and EasyEDA. Is this fine?

Reply:
Looks like this: https://workshops.hackclub.com/external-preview/karmanyaahm/onboard/pcb_level_1/